src/engraving/dom/note.cpp

```diff
@@ -976,10 +992,15 @@ SymId Note::noteHead() const
             }
         }
     }
+
+    if (scheme == NoteHeadScheme::HEAD_AUTO) {
+        scheme = accessibleScheme;
```

Comment:
Change this line to:
```C++
scheme = style().value(Sid::noteHeadScheme).value<NoteHeadScheme>();
```
Now you only look up the style value if you really need to.